GoldRush.js

```diff
@@ -0,0 +1,93 @@
+// const Matrix = require("./Matrix");
+
+class GoldRush extends Matrix {
+    constructor(rowNum, colNum) {
+        super(rowNum, colNum)
+        this.generateMatrix(rowNum, colNum)
+        this.addCoins(rowNum, colNum)
+        this.alter(0, 0, 1)
+        this.alter(4, 4, 2)
```

Comment:
This needs to be done dynamically based on the size of the matrix. It won't always be 5X5 no?